### PR TITLE
connectors: iceberg lakeformation sample config

### PR DESCRIPTION
**Pull request by bochenekmartin**

### CODE REVIEW

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc [hunk 1]
```diff
@@ -92,6 +92,89 @@ To enable Apache Kafka to retain the Debezium change event messages in their ori
...
+The connector can be used with AWS Glue catalog and AWS Lake Formation service.
+AWS Lake Formation allows to use assume role form of authentication only.
+Connectors UI does not support Lake Formation specific properties. Please, use JSON config editor instead.
+Sample config:
+
+```
+{
+    ...
+    "iceberg.catalog.client.assume-role.region": "the-region",
+    "iceberg.catalog.client.assume-role.arn": "arn:aws:iam::NNN:role/NNN",
```
Comment (tomasz-sadura) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]: what is the `NNN` here at the end? Are the two `NNN`s here the same?

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc [hunk 2]
```diff
@@ -92,6 +92,89 @@ To enable Apache Kafka to retain the Debezium change event messages in their ori
...
+    "iceberg.catalog.glue.account-id": "NNN",
+    "iceberg.catalog.catalog-impl": "org.apache.iceberg.aws.glue.GlueCatalog",
+    "iceberg.catalog.client.assume-role.tags.LakeFormationAuthorizedCaller": "iceberg-connector-1",
+    "iceberg.catalog.io-impl": "org.apache.iceberg.aws.s3.S3FileIO",
+    "iceberg.catalog": "catalog_name",
+    "iceberg.catalog.warehouse": "s3://NNN-region/my/data",
```
Comment (tomasz-sadura) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]: is the `NNN-region` just a bucket name, or it needs to be in this format? Above there is `CATALOG_WAREHOUSE=s3://bucket-name/`

**Review by tomasz-sadura — APPROVED** [review]

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc [hunk 3]
```diff
@@ -92,6 +92,89 @@ To enable Apache Kafka to retain the Debezium change event messages in their ori
 
 See also: https://debezium.io/documentation/reference/stable/transformations/event-flattening.html[Debezium New Record State Extraction SMT^]
 
+== Using Apache Spark with Iceberg
```
Comment (micheleRP) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]:
```suggestion
== Use Apache Spark with Iceberg
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc [hunk 4]
```diff
@@ -92,6 +92,89 @@ To enable Apache Kafka to retain the Debezium change event messages in their ori
...
+
+Data from Iceberg can be read by different systems like AWS Athena, Snowflake or Spark.
+The following example uses Iceberg REST catalog with S3 and Spark using the data stored in S3 bucket in Iceberg format.
```
Comment (micheleRP) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]:
```suggestion
Data from Iceberg can be read by different systems like AWS Athena, Snowflake, or Spark.
The following example uses Iceberg REST catalog with S3, and Spark uses the data stored in an S3 bucket in Iceberg format.
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc [hunk 5]
```diff
@@ -92,6 +92,89 @@ To enable Apache Kafka to retain the Debezium change event messages in their ori
...
+*
+```
+spark-sql ()> show tables in testnzss;
+foobarcouf
+```
+* Select data from table:
++
+```
+spark-sql ()> select * from testnzss.foobarcouf;
+```
```
Comment (micheleRP) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]:
```suggestion
Use Spark-SQL to:

* List databases:
+
```
spark-sql ()> show databases;
testnzss
```
* Show tables in database:
+
```
spark-sql ()> show tables in testnzss;
foobarcouf
```
* Select data from table:
+
```
spark-sql ()> select * from testnzss.foobarcouf;
```
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc [hunk 6]
```diff
@@ -92,6 +92,89 @@ To enable Apache Kafka to retain the Debezium change event messages in their ori
...
+
+== Using Iceberg connectors with AWS Glue catalog and AWS Lake Formation
```
Comment (micheleRP) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]:
```suggestion
== Use Iceberg connectors with AWS Glue catalog and AWS Lake Formation
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc [hunk 7]
```diff
@@ -92,6 +92,89 @@ To enable Apache Kafka to retain the Debezium change event messages in their ori
...
+The connector can be used with AWS Glue catalog and AWS Lake Formation service.
+AWS Lake Formation allows to use assume role form of authentication only.
+Connectors UI does not support Lake Formation specific properties. Please, use JSON config editor instead.
+Sample config:
```
Comment (micheleRP) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]:
```suggestion
The connector can be used with AWS Glue catalog and AWS Lake Formation service.
AWS Lake Formation only lets you use the role form of authentication.
The connectors UI does not support Lake Formation-specific properties. Use the JSON editor instead.
Sample configuration:
```

**Comment by micheleRP** [comment]
Thanks, @bochenekmartin! Would you please confirm the text suggestions I made?

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc [hunk 8]
```diff
@@ -268,6 +266,29 @@ testtable
 spark-sql ()> select * from testdb.testtable;
 ----
 
+== Use Iceberg connectors with AWS Glue catalog and AWS Lake Formation
```
Comment (micheleRP) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]:
```suggestion
== Use with AWS Glue Data Catalog and AWS Lake Formation
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc [hunk 9]
```diff
@@ -268,6 +266,29 @@ testtable
...
+
+The connector can be used with AWS Glue catalog and AWS Lake Formation service.
```
Comment (micheleRP) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]:
```suggestion
The connector can be used with the AWS Glue Data Catalog and the AWS Lake Formation service.
```

Comment (Feediver1) [reply modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]: active voice

Comment (Feediver1) [reply modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]: You can use the the connector with the AWS Glue Data Catalog and AWS Lake Formation service.

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc [hunk 10]
```diff
@@ -119,9 +119,7 @@ To enable Apache Kafka to retain the Debezium change event messages in their ori
 
 See also: https://debezium.io/documentation/reference/stable/transformations/event-flattening.html[Debezium New Record State Extraction SMT^]
 
-== Using analytical tools with Iceberg
-
-Data stored in Iceberg format can be read by different analytical systems like 
+== Use analytical tools with Iceberg
 
 The main use-case for Iceberg is a single storage of analytical data. It is cheap to read from different tools like AWS Athena, Snowflake or Apache Spark.
 The traditional way of importing data is pushing the data to every single tool. It means a higher cost of data transfer and storage.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]:
```suggestion
The primary use-case for Iceberg is for single storage of analytical data. It is cheap to read from different tools like AWS Athena, Snowflake or Apache Spark.
The traditional way of importing data is to push the data to every single tool, which means a higher cost of data transfer and storage.
```

**Review by Feediver1 — APPROVED** [review]
lgtm

**Pull request merged**